lib/client.js

```diff
@@ -266,11 +265,14 @@ extend(Raven.prototype, {
     kwargs.timestamp = new Date().toISOString().split('.')[0];
     kwargs.project = this.dsn.project_id;
     kwargs.platform = 'node';
+    kwargs.release = this.release;
 
-    // Only include release information if it is set
-    if (this.release) {
-      kwargs.release = this.release;
-    }
+    // Cleanup empty properties before sending them to the server
+    Object.keys(kwargs).forEach(function(key) {
+      if (typeof kwargs[key] === 'undefined' || kwargs[key] === '') {
```

Comment:
Why not just use `kwargs[key] === undefined`, the `typeof` check is only really needed for globals.

Reply:
I'm just used to use this check instead. It doesn't change any performance characteristics, as it's just one lookup. Although I think we should include `null` here as well, so we should end up with `kwargs[key] == null || kwargs[key] === ''` instead (allowing for `0` to be passed).